securitycenter/bigqueryv2/bigquery_test.go

```diff
@@ -0,0 +1,287 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bigqueryv2
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	securitycenter "cloud.google.com/go/securitycenter/apiv2"
+	"cloud.google.com/go/securitycenter/apiv2/securitycenterpb"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"github.com/google/uuid"
+)
+
+func orgID(t *testing.T) string {
+	orgID := os.Getenv("GCLOUD_ORGANIZATION")
+	if orgID == "" {
+		t.Fatal("GCLOUD_ORGANIZATION not set")
+	}
+	return orgID
+}
+
+func bigQueryDatasetName(t *testing.T) string {
+	bigQueryDatasetName := os.Getenv("SCC_BIGQUERY_DATASET_NAME")
```

Comment:
issue: hard-code the dataset name as a string literal into the samples. That way you don't need to pass in the name from the tests.

For tests that require a resource name, like delete/get/list/update, you can hard-code a demo dataset name here in the test that you build & teardown as part of the tests.

You might have your samples accept the `projectId` as a parameter and then construct a dataset in the sample.

Reply:
I get what you are saying but why do we have to hard-code the dataset name in the sample, when I am having a variable defined in the `systems_test.sh` so that whatever project the CI tries to run on, it can.